src/main/java/com/upsolver/datasources/jdbc/JDBCDataSource.java

```diff
@@ -1,6 +1,17 @@
 package com.upsolver.datasources.jdbc;
 
-import com.upsolver.common.datasources.*;
+import com.upsolver.common.datasources.DataLoader;
```

Comment:
Lots of import statements not needed, can you remove them?

Reply:
Imports are automatically optimized. What imports are not used?

Reply:
👍